README.md

```diff
@@ -31,35 +31,40 @@ SV2TTS is a three-stage deep learning framework that allows to create a numerica
 
 
 ## Setup
-Note: setup up this project is a lot of work. Somebody took the time to make [a better guide](https://poorlydocumented.com/2019/11/installing-corentinjs-real-time-voice-cloning-project-on-windows-10-from-scratch/) on how to install everything. I recommend using it. 
 
-### Requirements
-You will need the following whether you plan to use the toolbox only or to retrain the models.
+### 1. Install Requirements
 
-**Python 3.6+**.
+**Python 3.6 or 3.7** is needed to run the toolbox.
```

Comment:
What about 3.8?

Reply:
Python 3.8 will not work until tensorflow v2 is supported. See #401 .